source/scripting_v3/GTA.NaturalMotion/EuphoriaBase.cs

```diff
@@ -83,6 +102,83 @@ public void SendTo(Ped target, int duration)
 			SendTo(target);
 		}
 
+		/// <summary>
+		/// Sends message this NaturalMotion behavior on the <see cref="Ped"/>, but don't start a new ragdoll task so the <see cref="Ped"/> can have multiple NM behaviors.
+		/// You'll need to start a ragdoll task with <see cref="Ped.Ragdoll(int, RagdollType)"/> before the <see cref="Ped"/> can recieve this <see cref="Message"/>.
+		/// </summary>
+		/// <remarks>
+		/// Although <see cref="SendTo(Ped)"/> wouldn't create a new ragdoll task if the <see cref="Ped"/> runs a script ragdoll tasks in most cases,
+		/// this method guarantees it sends the message if the <see cref="Ped"/> can receive NM messages but doesn't create a new ragdoll task.
+		/// </remarks>
+		/// <param name="target">The <see cref="Ped"/> to send the <see cref="Message"/> to.</param>
+		/// <param name="startBehavior">Specifies whether this message should start from the beginning.</param>
+		/// <param name="ifNMScriptControlRunning">
+		/// <para>Specifies whether the messages should be sent to the only if they <see cref="Ped"/> has a <c>CTaskNMScriptControl</c> running, which can be started with <see cref="Ped.Ragdoll(int, RagdollType)"/>.</para>
+		/// <para>
+		/// If you set this parameter to <see langword="false"/> and you need to set the <see cref="Ped"/> from a ragdoll state to an animated state after a certain timeout (in game time),
+		/// you should call <see cref="Ped.CancelRagdoll()"/> on them after a certain timeout if they are alive and <see cref="Ped.IsRagdoll"/> returns <see langword="true"/>.
+		/// </para>
+		/// </param>
+		public void SendToNoNewRagdollTask(Ped target, bool startBehavior = true, bool ifNMScriptControlRunning = true)
+		{
+			if (startBehavior)
+			{
+				SendMessagesWithStartParamEnabledToInternal(target, ifNMScriptControlRunning);
+			}
+			else
+			{
+				SendMessagesWithoutExplicitStartParamToInternal(target, ifNMScriptControlRunning);
+			}
+		}
+
+		public void SendMessagesWithStartParamEnabledToInternal(Ped target, bool ifNMScriptControlRunning = true)
+		{
+			// Use a cloned dictionary so this message instance can keep the original parameters
+			// Shallow copying should be enough
+			var newBoolIntFloatArguments = GetClonedDictWithStartParamEnabled();
```

Comment:
The old way of passing the dictionaries to `SHVDN.NativeMemory` directly is probably more efficient. I don't see the need for a clone (assuming `start` is set in the message helper itself), since `SHVDN.NativeMemory.SendEuphoriaMessage` is/was synchronous.